app.js

```diff
@@ -13,22 +13,31 @@ var config = {
 
 require('./index.js').connect(function (context) {
 
+    context.fs.readdir(context.path.join(__dirname, 'api', 'models'), (err, files) => {
+        //Load ALL modules.
+        //Don't get confused by weird logging order! console.log is async and will mix up the actual loading order.
+        files.forEach(file => {
+            context.component('models').module(file.replace('.js',''));
```

Comment:
We are not actually loading all models in index.js. See the log when running the current develop version: Do you see all models in the models directory appear as loaded in the log? No.
This is because in index.js we don't actually load all models. 
We create a function that will either: 

- Load a model and cache it

- Return a previously loaded, cached model

This way, if we don't call for example 
`context.component('models').module('relationshipTypes'')` somewhere, this model will never be actually loaded.

By explicitly loading all models by default in app.js we assure that every model is loaded and we don't need to create a controller, which creates a dsap, which loads the model for each model.